libevmasm/Assembly.cpp

```diff
@@ -74,7 +74,7 @@ unsigned Assembly::codeSize(unsigned subTagSize) const
 			ret += i.second.size();
 
 		for (AssemblyItem const& i: m_items)
-			ret += i.bytesRequired(tagSize, m_evmVersion, Precision::Approximate);
+			ret += i.bytesRequired(tagSize, m_evmVersion, Precision::Precise);
```

Comment:
I didn't remove the only use of `Precision::Approximate` inside `bytesRequired` because there's a path code in which it is called by the `PeepholeOptimizer` before `Assembly::assemble`. In that case, there is no register occurrences of immutables which `Precision::Approximate` handled by assuming a minimum of one. 
I checked and `Assembly::codeSize` is only used one time and that seems not to be in any problematic path code such the one described before. So I think it is safe to make this change here.

cc @ekpyron